cli/internal/cloudcmd/tfupgrade.go

```diff
@@ -52,20 +51,20 @@ func planUpgrade(
 	return hasDiff, nil
 }
 
-// moveUpgradeToCurrent replaces the an existing Terraform workspace with a workspace holding migrated Terraform resources.
-func moveUpgradeToCurrent(fileHandler file.Handler, existingWorkspace, upgradeWorkingDir string) error {
-	if err := fileHandler.RemoveAll(existingWorkspace); err != nil {
-		return fmt.Errorf("removing old terraform directory: %w", err)
+// rollbackToBackup replaces the existing Terraform workspace with the backup.
+func rollbackToBackup(fileHandler file.Handler, workingDir, backupDir string) error {
```

Comment:
```suggestion
// restoreBackup replaces the existing Terraform workspace with the backup.
func restoreBackup(fileHandler file.Handler, workingDir, backupDir string) error {
```
naming nit: We don't actually "rollback" any changes in case things go wrong, and this names reads a bit easier imo